src/batou_ext/oci.py

```diff
@@ -135,7 +141,11 @@ def configure(self):
             ),
         )
 
+        if self.rebuild:
+            self += batou_ext.nix.Rebuild()
+
     def verify(self):
+        self._need_restart = False
         self.assert_no_changes()
         self.envfile.assert_no_changes()
```

Comment:
the container needs to be restarted when the envfile changes, nixos will not do that on rebuild since the templated module does not change because the path to the envfile is a string and therefore will only trigger a restart from a rebuild when it's path changes regardless of its contents